api/filter/tcp_proxy.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package envoy.api.v2.filter;
+
+import "google/protobuf/duration.proto";
+
+// [V2-API-DIFF] The route match now takes place in the FilterChainMatch table.
+message TcpProxy {
+  // The human readable prefix to use when emitting statistics for the
+  // TCP proxy filter. See the statistics documentation for more information.
+  string stat_prefix = 1;
+
+  // The upstream cluster to connect to.
+  string cluster = 2;
```

Comment:
Yes, for long living connections the utility is small. Nevertheless, some application protocols might not have dedicated filters in envoy yet (think smth like redis), where this might be useful.